Sources/Model/Paging/HyperwalletTransferMethodQueryParam.swift

```diff
@@ -55,47 +62,29 @@ public class HyperwalletTransferMethodPagination: HyperwalletPagination {
         case descendantStatus = "-status"
     }
 
-    /// Builds the `HyperwalletTransferMethodPagination`'s URL Queries
-    ///
-    /// - Returns: Returns a dictionary of URL Query
     override public func toQuery() -> [String: String] {
         var query = super.toQuery()
-
         if let date = createdAfter {
-            query["createdAfter"] = HyperwalletTransferMethodPagination.iso8601.string(from: date)
+            query[QueryParam.createdAfter.rawValue] = HyperwalletTransferMethodQueryParam.iso8601.string(from: date)
         }
-
         if let date = createdBefore {
-            query["createdBefore"] = HyperwalletTransferMethodPagination.iso8601.string(from: date)
+            query[QueryParam.createdBefore.rawValue] = HyperwalletTransferMethodQueryParam.iso8601.string(from: date)
         }
-
-        if status != nil {
-            query["status"] = status?.rawValue
+        if let status = status {
+            query[QueryParam.status.rawValue] = status.rawValue
         }
-
-        if sortBy != nil {
-            query["sortBy"] = sortBy?.rawValue
+        if let sortBy = sortBy {
+            query[QueryParam.sortBy.rawValue] = sortBy.rawValue
         }
-
         return query
     }
 
-    /// Returns DateFormatter based at ISO860-1
-    ///
-    /// Example to convert Date to String:
-    ///     let date = Date()
-    ///     let dateFormatted = iso8601.string(from: date)
-    ///     print(dateFormatted) // 2019-01-23T03:04:13.959Z
-    ///
-    /// Example to convert Date to String:
-    ///     let value = "2019-01-23T03:04:13.959Z"
-    ///     let date = iso8601.date(from: value)
-    public static let iso8601: DateFormatter = {
+    static let iso8601: DateFormatter = {
```

Comment:
Remove this from here and use the Date Extension

Reply:
fixed